cmake/ThirdParty.cmake

```diff
@@ -114,36 +93,16 @@ if (OPENASSETIO_ENABLE_PYTHON)
     #-------------------------------------------------------------------
     # Target to create a Python virtual environment in the install tree.
 
-    # Download core setuptools dependencies so we don't always need an
-    # online connection to local install OpenAssetIO. Need to cache the
-    # build tools explicitly due to build isolation, see:
-    # https://pip.pypa.io/en/stable/reference/build-system/pyproject-toml/#build-process
-    # This also includes runtime dependencies of packages in the
-    # resources directory, whose install will otherwise fail due to
-    # --no-index.
-    openassetio_add_pip_download_target(
-        openassetio.internal.python-venv.download-deps
-        "Downloading core Python libraries"
-        wheel==0.37.1 setuptools==49.0
-        # We must upgrade pip so that the build directory is not
-        # copied to a temporary directory, since doing so could cause
-        # Windows to fail with permissions errors when concurrently
-        # installing Python packages and building the C++ component.
```

Comment:
[fix] I _think_ this comment block should move to the (new) L105? As it seems unrelated to offline support, and the actual update is done when we make the venv? 

(Also, whilst I remember, does this mean we need to document the above when using your own python env?)

Reply:
I don't _think_ we have this problem any more, since we're no longer `pip install`ing our projects. Technically we could remove the `pip` upgrade, but it's still nice to pin versions to avoid nasty surprises.

Reply:
Cool, we should keep it in mind when we add pip installing for tests back in again :)